python312Packages.jupyter-server-fileid: 0.9.2 -> 0.9.3

r-ryantm opened the pull request:
Automatic update generated by [nixpkgs-update](https://github.com/ryantm/nixpkgs-update) tools. This update was made based on information from passthru.updateScript.


meta.description for python312Packages.jupyter-server-fileid is: Extension that maintains file IDs for documents in a running Jupyter Server

meta.homepage for python312Packages.jupyter-server-fileid is: https://github.com/jupyter-server/jupyter_server_fileid

meta.changelog for python312Packages.jupyter-server-fileid is: https://github.com/jupyter-server/jupyter_server_fileid/blob/refs/tags/v0.9.3/CHANGELOG.md


###### Updates performed
- Ran passthru.UpdateScript

###### To inspect upstream changes





###### Impact

<b>Checks done</b>

---

- built on NixOS
- The tests defined in `passthru.tests`, if any, passed
- found 0.9.3 with grep in /nix/store/jv7ncriz2l4cla095hjjrpf6wqg94haf-python3.12-jupyter-server-fileid-0.9.3
- found 0.9.3 in filename of file in /nix/store/jv7ncriz2l4cla095hjjrpf6wqg94haf-python3.12-jupyter-server-fileid-0.9.3

---

<details>
<summary>
<b>Rebuild report</b> (if merged into master) (click to expand)
</summary>

```
5 total rebuild path(s)

5 package rebuild(s)

First fifty rebuilds by attrpath

python311Packages.jupyter-collaboration
python311Packages.jupyter-server-fileid
python312Packages.jupyter-collaboration
python312Packages.jupyter-server-fileid
```

</details>

<details>
<summary>
<b>Instructions to test this update</b> (click to expand)
</summary>

---

Either **download from Cachix**:
```
nix-store -r /nix/store/jv7ncriz2l4cla095hjjrpf6wqg94haf-python3.12-jupyter-server-fileid-0.9.3 \
  --option binary-caches 'https://cache.nixos.org/ https://nix-community.cachix.org/' \
  --option trusted-public-keys '
  nix-community.cachix.org-1:mB9FSh9qf2dCimDSUo8Zy7bkq5CX+/rkCWyvRCYg3Fs=
  cache.nixos.org-1:6NCHdD59X431o0gWypbMrAURkbJ16ZPMQFGspcDShjY=
  '
```
(The Cachix cache is only trusted for this store-path realization.)
For the Cachix download to work, your user must be in the `trusted-users` list or you can use `sudo` since root is effectively trusted.

Or, **build yourself**:
```
nix-build -A python312Packages.jupyter-server-fileid https://github.com/r-ryantm/nixpkgs/archive/3a57c09665bf6e7ee4f9519811b8d998329cd6ae.tar.gz
```
Or:
```
nix build github:r-ryantm/nixpkgs/3a57c09665bf6e7ee4f9519811b8d998329cd6ae#python312Packages.jupyter-server-fileid
```

After you've downloaded or built it, look at the files and if there are any, run the binaries:
```
ls -la /nix/store/jv7ncriz2l4cla095hjjrpf6wqg94haf-python3.12-jupyter-server-fileid-0.9.3
ls -la /nix/store/jv7ncriz2l4cla095hjjrpf6wqg94haf-python3.12-jupyter-server-fileid-0.9.3/bin
```

---

</details>
<br/>



### Pre-merge build results

We have automatically built all packages that will get rebuilt due to
this change.

This gives evidence on whether the upgrade will break dependent packages.
Note sometimes packages show up as _failed to build_ independent of the
change, simply because they are already broken on the target branch.

Result of `nixpkgs-review` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>8 packages built:</summary>
  <ul>
    <li>python311Packages.jupyter-collaboration</li>
    <li>python311Packages.jupyter-collaboration.dist</li>
    <li>python311Packages.jupyter-server-fileid</li>
    <li>python311Packages.jupyter-server-fileid.dist</li>
    <li>python312Packages.jupyter-collaboration</li>
    <li>python312Packages.jupyter-collaboration.dist</li>
    <li>python312Packages.jupyter-server-fileid</li>
    <li>python312Packages.jupyter-server-fileid.dist</li>
  </ul>
</details>

---

###### Maintainer pings

cc @dotlambda for [testing](https://github.com/ryantm/nixpkgs-update/blob/main/doc/nixpkgs-maintainer-faq.md#r-ryantm-opened-a-pr-for-my-package-what-do-i-do).

> [!TIP]
> As a maintainer, if your package is located under `pkgs/by-name/*`, you can comment **`@NixOS/nixpkgs-merge-bot merge`** to automatically merge this update using the [`nixpkgs-merge-bot`](https://github.com/NixOS/nixpkgs-merge-bot).

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

dotlambda commented on pkgs/development/python-modules/jupyter-server-fileid/default.nix:
```diff
@@ -12,7 +12,7 @@
   pname = "jupyter-server-fileid";
-  version = "0.9.2";
+  version = "0.9.3";
 
```
```suggestion
  pyproject = true;

```

dotlambda commented on pkgs/development/python-modules/jupyter-server-fileid/default.nix:
```diff
@@ -22,7 +22,7 @@ buildPythonPackage rec {
+    hash = "sha256-ob7hnqU7GdaDHEPF7+gwkmsboKZgiiLzzwxbBUwYHYo=";
   };
 
   nativeBuildInputs = [ hatchling ];
```
```suggestion
  build-system = [ hatchling ];
```

dotlambda committed:
python312Packages.jupyter-server-fileid: 0.9.2 -> 0.9.3

Diff: https://github.com/jupyter-server/jupyter_server_fileid/compare/refs/tags/v0.9.2...v0.9.3

Changelog: https://github.com/jupyter-server/jupyter_server_fileid/blob/refs/tags/v0.9.3/CHANGELOG.md

dotlambda commented:
Result of `nixpkgs-review pr 340155` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>8 packages built:</summary>
  <ul>
    <li>python311Packages.jupyter-collaboration</li>
    <li>python311Packages.jupyter-collaboration.dist</li>
    <li>python311Packages.jupyter-server-fileid</li>
    <li>python311Packages.jupyter-server-fileid.dist</li>
    <li>python312Packages.jupyter-collaboration</li>
    <li>python312Packages.jupyter-collaboration.dist</li>
    <li>python312Packages.jupyter-server-fileid</li>
    <li>python312Packages.jupyter-server-fileid.dist</li>
  </ul>
</details>
